.github/workflows/integration_test_run.yaml

```diff
@@ -329,3 +299,23 @@ jobs:
           target: ${{ inputs.zap-target-protocol }}://${{ env.ZAP_TARGET }}:${{ inputs.zap-target-port }}/
           cmd_options: ${{ inputs.zap-cmd-options }}
           rules_file_name: ${{ inputs.zap-rules-file-name }}
+      - name: Pack charm
+        if: always()
+        working-directory: ${{ inputs.working-directory }}
+        run: |
+          # Not ideal, but the charm doesn't get always packed during the integration tests step
```

Comment:
I think this comment can now be gone :D

Reply:
Not really, no. Running the integration tests successfully is expected to leave a packed charm. However, in case of failure this doesn't happen